api/anomaly/services/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .slack import SlackAlert
 from .detectionRules import DetectionRules
 from .rootCauseAnalyses import RootCauseAnalyses
+from .plotly import Plotly
```

Comment:
Please rename file from `plotly` to `PlotChart` or similar, as plotly is the library so it'll be confusing and might cause problems.